.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
                 - GROUP1="$UPDATE_SUBGROUP1 $(find test/functional/{autoupdate,bundleinfo,verify-legacy} -name *.bats -printf '%p ')"
                 - GROUP2="$UPDATE_SUBGROUP2 $(find test/functional/{checkupdate,hashdump,mirror,usability} -name *.bats \( ! -name usa-config-file.bats \) -printf '%p ')"
                 - GROUP3="$(find test/functional/{diagnose,search,os-install,repair} -name *.bats -printf '%p ')"
-                - GROUP4="$(find test/functional/{bundleadd,bundleremove,bundlelist,signature} -name *.bats -printf '%p ')"
+                - GROUP4="$(find test/functional/{bundleadd,bundleremove,bundlelist,signature,3rd-party} -name *.bats -printf '%p ')"
```

Comment:
> I'm hesitant to have all 3rd-party test under the same 3rd-party directory. I would prefer to start with keeping each 3rd-party command on its own directory for now. So in this case you would need three directories: 3rd_party_add, 3rd_party_remove and 3rd_party_list. Also remember we want to eventually migrate all directory names to use snake case, so use `_` in the directory name instead of `-`.

Ah..I thought about that too, The reason I went with this approach is because all three commands are helpful together. Thats how a user would use it and think of them as some thing called **3rd-party repo manage**. Any specific reason why you wanted a folder for each? Will make to change all to snake case

Reply:
> Finally, you forgot to update the description of the job in line 39.

Okay, I will look at #1193 and try to use same approach